tests/test_arcaflow_plugin_stressng.py

```diff
@@ -53,15 +56,56 @@ def test_serialization():
             )
         )
 
+        plugin.test_object_serialization(
+            stressng_schema.MmapStressorParams(
+                stressor=stressng_schema.Stressors.MMAP,
+                workers=2,
+                mmap_bytes="20%",
+                mmap_async=True,
+            )
+        )
+
         plugin.test_object_serialization(
             stressng_schema.MatrixStressorParams(
-                stressor=stressng_schema.Stressors.MATRIX, workers=2
+                stressor=stressng_schema.Stressors.MATRIX,
+                workers=2,
+                matrix_method=stressng_schema.MatrixMethod.ADD,
             )
         )
 
         plugin.test_object_serialization(
             stressng_schema.MqStressorParams(
-                stressor=stressng_schema.Stressors.MQ, workers=2
+                stressor=stressng_schema.Stressors.MQ,
+                workers=2,
+                mq_ops=4,
+            )
+        )
+
+        plugin.test_object_serialization(
+            stressng_schema.HDDStressorParams(
+                stressor=stressng_schema.Stressors.HDD,
+                workers=2,
+                hdd_opts=[
+                    stressng_schema.HddOpts.DIRECT,
+                    stressng_schema.HddOpts.SYNC,
+                ],
+            )
+        )
```

Comment:
Consider listing the cases in order -- it makes it easier to see when something's missing.

Reply:
What order? Again as above, the order here matches the order the items are defined in the schema. This order is actually helpful as I can review the list of defined stressors in the schema and confirm in that order that I have tests defined. Re-ordering would need to be done in 4-5 different places for consistency.